terraform-infra-approvals/pre-shared-infrastructure.json

```diff
@@ -16,7 +16,10 @@
     {"type": "azurerm_virtual_network_peering"},
     {"type": "azurerm_media_transform"},
     {"type": "azurerm_role_assignment"}
-    
+ 
   ],
-  "module_calls": []
+  
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/cnp-module-key-vault?ref=kv_networkacls"}
```

Comment:
can you open a PR on the module first so we can discuss the change there?

You can still test this in sbox with no restrictions